server/mailet/mock-smtp-server/src/resources/logback.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+                <encoder>
+                        <pattern>%d{HH:mm:ss.SSS} %highlight([%-5level]) %logger{15} - %msg%n%rEx</pattern>
+                        <immediateFlush>false</immediateFlush>
+                </encoder>
+        </appender>
+
+        <root level="WARN">
```

Comment:
I think you want `DEBUG` for a mocking server

Reply:
I want DEBUG for the class of the server, and the main mail dependency, and it is exactly what

```
        <logger name="org.apache.james.mock.smtp.server" level="DEBUG"/>
        <logger name="org.subethamail.smtp" level="DEBUG"/>
```